src/resolve/resolved-mdns.c

```diff
@@ -359,6 +359,10 @@ static int mdns_goodbye_callback(sd_event_source *s, uint64_t usec, void *userda
 
         dns_cache_prune(&scope->cache);
 
+        r = mdns_notify_browsers_goodbye(scope);
+        if (r < 0)
+                log_error_errno(r, "mDNS: Failed to notify service subscribers of goodbyes, %m");
```

Comment:
we usually use `: %m` rather than `, %m`. here and everywhere.

and also, as mentioned elsewhere: please downgrade to log_warning_errno() for errors you ignore, and suffix the log message with ", ignoring: %m" to make this clear. it's ok if you ignore errors, but the log messages should make this clear